app/views/handlers/individual_response.py

```diff
@@ -155,9 +160,12 @@ def get_context(self):
     def _publish_fulfilment_request(self, mobile_number=None):
         topic_id = current_app.config["EQ_FULFILMENT_TOPIC_ID"]
         fulfilment_request = FulfilmentRequest(self._metadata, mobile_number)
-        return current_app.eq["publisher"].publish(
-            topic_id, message=fulfilment_request.payload
-        )
+        try:
+            return current_app.eq["publisher"].publish(
+                topic_id, message=fulfilment_request.payload
+            )
+        except PublicationFailed:
+            raise FulfilmentRequestFailedException
```

Comment:
We already raise our custom `PublicationFailed`, so why do we raise another custom Exception?. We could just make the handler support `PublicationFailed` then this try expect isn't needed. Thoughts?

Reply:
Because I didn't want to use the generic PublicationFailed to execute Individual Response code. It seems to me that the pattern for publish should be Publish -> Error -> Raise PublicationFailed, allow the calling method to handle that in it's own way. Any future pub sub methods aren't then tied to the Individual Response fulfilment requests

Reply:
Ya good point. 👍